com.unity.render-pipelines.core/Editor/Lighting/SRPLensFlareElementEditor.cs

```diff
@@ -0,0 +1,490 @@
+                        {
+                            lensFlareProp.objectReferenceValue = tmpTex;
+                            lensFlareProp.serializedObject.ApplyModifiedProperties();
+                        }
+                    }
+                    else if (newType == SRPLensFlareType.Circle || newType == SRPLensFlareType.Polygon)
+                    {
+                        SerializedProperty fallOffProp = property.FindPropertyRelative("fallOff");
+                        SerializedProperty edgeOffsetProp = property.FindPropertyRelative("edgeOffset");
+                        SerializedProperty sdfRoundnessProp = property.FindPropertyRelative("sdfRoundness");
+                        SerializedProperty sideCountProp = property.FindPropertyRelative("sideCount");
+                        SerializedProperty inverseSDFProp = property.FindPropertyRelative("inverseSDF");
+
+                        rect = GetNextRect();
+                        if ((tmp = EditorGUI.Slider(rect, Styles.edgeOffset, edgeOffsetProp.floatValue, 0.0f, 1.0f)) != edgeOffsetProp.floatValue)
+                            edgeOffsetProp.floatValue = Mathf.Clamp01(tmp);
+
+                        rect = GetNextRect();
+                        //if ((tmp = EditorGUI.FloatField(rect, Styles.fallOff, fallOffProp.floatValue)) != fallOffProp.floatValue)
+                        if ((tmp = EditorGUI.Slider(rect, Styles.fallOff, fallOffProp.floatValue, 0.0f, 1.0f)) != fallOffProp.floatValue)
+                            fallOffProp.floatValue = Mathf.Max(tmp, 0.0f);
+
+                        if (newType == SRPLensFlareType.Polygon)
+                        {
+                            rect = GetNextRect();
+                            if ((tmp = EditorGUI.IntSlider(rect, Styles.sideCount, sideCountProp.intValue, 3, 32)) != sideCountProp.intValue)
+                                sideCountProp.intValue = (int)Mathf.Max(tmp, 0);
+
+                            rect = GetNextRect();
+                            if ((tmp = EditorGUI.Slider(rect, Styles.sdfRoundness, sdfRoundnessProp.floatValue, 0.0f, 1.0f)) != sdfRoundnessProp.floatValue)
+                                sdfRoundnessProp.floatValue = Mathf.Clamp01(tmp);
+                        }
+
+                        rect = GetNextRect();
+                        if ((tmpBool = EditorGUI.Toggle(rect, Styles.inverseSDF, inverseSDFProp.boolValue)) != inverseSDFProp.boolValue)
+                            inverseSDFProp.boolValue = tmpBool;
+                    }
+
+                    rect = GetNextRect();
+                    if ((tmpBool = EditorGUI.Toggle(rect, Styles.preserveAspectRatio, preserveAspectRatioProp.boolValue)) != preserveAspectRatioProp.boolValue)
+                        preserveAspectRatioProp.boolValue = tmpBool;
+                }
+
+                rect = GetNextRect();
+                EditorGUI.TextArea(rect, "Color", EditorStyles.boldLabel);
```

Comment:
label should be in a style class to not search them in the logic and it should be localizable.
Please use either **L18n** or **EditorGUIUtility.TrTextContent**